elements/pfe-sass/maps/_typography.scss

```diff
@@ -0,0 +1,68 @@
+$pfe-typography-base: (
+  font-family:          '"Red Hat Text", "RedHatText", "Overpass", Overpass, Arial, sans-serif',
+  font-family--heading: '"Red Hat Display", "RedHatDisplay", "Overpass", Overpass, Arial, sans-serif',
+  font-family--code:    '"Overpass Mono", Consolas, Monaco, "Andale Mono", monospace',
+  line-height--heading:      1.2,
+  line-height:               1.3,
+  font-weight--light:        400,
+  font-weight--normal:       500,
+  font-weight--semi-bold:    700,
+  font-weight--bold:         900,
+);
+
+
+////////////////////////////////////////////////
+// Global typography sizing as t-shirt sizes 
+// Values are multiplying base font size vars to allow for scaling on mobile 
+
+// Example: --pf-global--FontSize--4xl
+
+$pf-type-sizing: (
+   FontSize--root:  1em, // for use on the HTML or root element
```

Comment:
should we keep the original variable of just `font-size` for the root?
https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-sass/maps/_general.scss#L6